index.html

```diff
@@ -3,19 +3,37 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | AnnaFYZ</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
     <div id="root">
+      <div>
+        
+      </div>
+      <div id="searchBar">
+        <select id="episode-selector" class="form-control">
```

Comment:
nice!!! I have done in similar way too but someone told me that its better to create select element in javaScript than html. it helps for debugging.